docs/dependencies.html

```diff
@@ -104,3 +104,84 @@ <h2>Comparison to other systems</h2>
   tool, but again only outputs a single package at a time. Writing up the dependencies can be
   eased by using something like <code>go list -f '{{.Deps}}' &lt;package&gt;</code> to discover
   all the dependencies for the package in question.</p>
+
+<h2>Verification</h2>
```

Comment:
I know you can add hashes to all rules so it's not specifically related to third party stuff however these seem closely related. Feels like a better home for this docco.